tetris.py

```diff
@@ -27,18 +28,41 @@ def run_gui():
     root = tkinter.Tk()
     root.resizable(False, False)
 
-    topbar = tkinter.Label(root)
-    topbar['bg'] = D_GREY
-    topbar.grid(row=0, column=0, sticky='we')
     game_canvas = tkinter.Canvas(
         root,
         width=square_size * game_width,
         height=square_size * game_height,
         highlightthickness=0,
+        borderwidth=1
     )
-    game_canvas.grid()
+    game_canvas.grid(row=1, sticky='nswe')
 
-    tetris_gui = TetrisGUI(game_speed, game_canvas)
+    topbar = tkinter.Frame(root, bg=D_GREY, height=square_size*2, relief = 'ridge')
+    topbar.grid(row=0, columnspan=2, sticky='we')
+    topbar.columnconfigure(0, weight=1)
+
+    tetris_gui = TetrisGUI(game_speed, game_canvas, topbar_next_block)
+
+    topbar_time = tkinter.Label(topbar, bg=D_GREY, text=tetris_gui.timer(), font = 'digital-7', fg='orange', borderwidth=1)
+    topbar_time.pack(side='left', padx=10)
+
+    topbar_score = tkinter.Label(topbar, bg=D_GREY, text='foo', font = 'digital-7', fg='orange', borderwidth=1)
+    topbar_score.pack(side='left', fill='x', expand=True)
+
+    topbar_next_block = tkinter.Canvas(topbar, bg=D_GREY, width = square_size*4, height = square_size*2)
+    topbar_next_block.pack(side='right')
+
+    sidebar = tkinter.Frame(root, bg=D_GREY, height=square_size*game_height)
```

Comment:
What happens if you get rid of height, like this?

```suggestion
    sidebar = tkinter.Frame(root, bg=D_GREY)
```